src/main/resources/modules/uti/telemed_path.json

```
@@ -0,0 +1,535 @@
+{
+  "name": "UTI_Telemed",
```

Similar note as above
```suggestion
  "name": "UTI_Telemed_Path",
```